.github/workflows/kubeconform-validation.yml

```diff
@@ -41,7 +43,17 @@ jobs:
         with:
           kubectl: latest
           kustomize: latest
-          kubeconform: latest
+          kubeconform: v0.6.6
+
+      - name: Print versions
+        run: |
+          echo "Kubectl version installed:"
+          kubectl version --client
+          echo "Kustomize version installed:"
+          kustomize version
+          echo "Kubeconform version installed:"
+          kubeconform -v
+
       - name: Check k8s manifests
         run: |
-          kustomize build $TUTOR_ROOT/env | kubeconform -strict -ignore-missing-schemas -kubernetes-version latest
+          kustomize build $TUTOR_ROOT/env | kubeconform -strict -ignore-missing-schemas -kubernetes-version master
```

Comment:
I had a question I just recalled: what does it mean the "master" version of Kubernetes? latest Kubernetes tag? Kubernetes master branch?

Reply:
@jfavellar90  master refers to master of the kubernetes repository, it is also possible to put a specific tag and not reference direct to master